bigbluebutton-html5/private/locales/en.json

```diff
@@ -265,5 +265,32 @@
     "app.toast.chat.singular":"you have {0} new message in {1}",
     "app.toast.chat.plural":"you have {0} new messages in {1}",
     "app.notification.recordingStart": "This session is now being recorded",
-    "app.notification.recordingStop": "This session is not being recorded anymore"
+    "app.notification.recordingStop": "This session is not being recorded anymore",
+    "app.whiteboard.toolbar.tools": "Tools",
+    "app.whiteboard.toolbar.tools.hand": "Hand",
+    "app.whiteboard.toolbar.tools.pencil": "Pencil",
+    "app.whiteboard.toolbar.tools.rectangle": "Rectangle",
+    "app.whiteboard.toolbar.tools.triangle": "Triangle",
+    "app.whiteboard.toolbar.tools.ellipse": "Ellipse",
+    "app.whiteboard.toolbar.tools.line": "Line",
+    "app.whiteboard.toolbar.tools.text": "Text",
+    "app.whiteboard.toolbar.thickness": "Thickness List",
+    "app.whiteboard.toolbar.color": "Color List",
+    "app.whiteboard.toolbar.color.black": "Black",
+    "app.whiteboard.toolbar.color.white": "White",
+    "app.whiteboard.toolbar.color.red": "Red",
+    "app.whiteboard.toolbar.color.orange": "Orange",
+    "app.whiteboard.toolbar.color.eletricLime": "Eletric Lime",
```

Comment:
I'm curious to see the translation of this string in other languages. Personally I'd be more in favor of a simpler name like ... "Yellow"

Reply:
Maybe we can change to use only basic colors in color list.

Reply:
The colors were copied from the [Flash client's list](https://github.com/bigbluebutton/bigbluebutton/blob/v2.0.x-release/bigbluebutton-client/src/org/bigbluebutton/modules/whiteboard/views/WhiteboardToolbar.mxml#L83-L84) of colors..